ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.txt

```diff
@@ -9,295 +9,295 @@ hideall "--sync--"
-4335.2 "Verdant Tempest" sync /:Trinity Seeker:5AD3:/
-4346.5 "Merciful Arc" sync /:Trinity Seeker:5AD4:/
-4349.7 "--middle--" sync /:Trinity Seeker:5A9A:/
-4354.0 "Manifest Avatar" sync /:Trinity Seeker:5ADA:/
-4361.2 "First Mercy" sync /:Trinity Seeker:5B61:/
-4364.4 "Second Mercy" sync /:Trinity Seeker:5B62:/
-4365.4 "--jump--" sync /:Seeker Avatar:5A9A:/
-4367.6 "Third Mercy" sync /:Trinity Seeker:5B63:/
-4370.8 "Fourth Mercy" sync /:Trinity Seeker:5B64:/
-4371.6 "Iron Splitter" sync /:Seeker Avatar:5AC0:/
-4373.2 "Mercy Fourfold 1" #sync /:Trinity Seeker:5B94:/
-4375.1 "Mercy Fourfold 2" #sync /:Trinity Seeker:5B94:/
-4377.0 "Mercy Fourfold 3" #sync /:Trinity Seeker:5B94:/
-4378.9 "Mercy Fourfold 4" #sync /:Trinity Seeker:5B94:/
-4380.6 "Iron Splitter" sync /:Seeker Avatar:5AC0:/
-4382.8 "--jump--" sync /:Seeker Avatar:5A9A:/
-4389.5 "Iron Splitter" sync /:Seeker Avatar:5AC0:/
-4390.3 "Seasons Of Mercy" sync /:Trinity Seeker:5AC7:/
-4394.8 "Merciful Breeze" sync /:Trinity Seeker:5AC8:/
-4396.3 "Merciful Moon" sync /:Aetherial Orb:5AC9:/
-4398.9 "Merciful Breeze" sync /:Trinity Seeker:5AC8:/
-4400.9 "Iron Rose" sync /:Seeker Avatar:5AD9:/
-4402.2 "Merciful Blooms" sync /:Trinity Seeker:5ACA:/
-4412.3 "Verdant Tempest" sync /:Trinity Seeker:5AD3:/
+4262.8 "--sync--" sync / 14:Trinity Seeker:5A97:/ window 300,10
+4265.8 "Verdant Path" sync / 1[56]:Trinity Seeker:5A97:/
+4270.4 "Act Of Mercy" sync / 1[56]:Trinity Seeker:5ACF:/
+4276.5 "--middle--" sync / 1[56]:Trinity Seeker:5A9A:/
+4280.8 "Manifest Avatar" sync / 1[56]:Trinity Seeker:5ADA:/
+4286.9 "Baleful Comet 1" #sync / 1[56]:Seeker Avatar:5AD7:/
+4287.9 "Baleful Comet 2" #sync / 1[56]:Seeker Avatar:5AD7:/
+4288.9 "Baleful Comet 3" #sync / 1[56]: Seeker Avatar:5AD7:/
+4289.9 "Baleful Comet 4" #sync / 1[56]:Seeker Avatar:5AD7:/
+4290.9 "--middle--" sync / 1[56]:Trinity Seeker:5A9A:/
+4294.5 "Baleful Firestorm" #sync / 1[56]:Seeker Avatar:5AD8:/
+4295.1 "First Mercy" sync / 1[56]:Trinity Seeker:5B61:/
+4296.5 "Baleful Firestorm" #sync / 1[56]:Seeker Avatar:5AD8:/
+4298.3 "Second Mercy" sync / 1[56]:Trinity Seeker:5B62:/
+4298.5 "Baleful Firestorm" #sync / 1[56]:Seeker Avatar:5AD8:/
+4300.5 "Baleful Firestorm" #sync / 1[56]:Seeker Avatar:5AD8:/
+4301.5 "Third Mercy" sync / 1[56]:Trinity Seeker:5B63:/
+4304.7 "Fourth Mercy" sync / 1[56]:Trinity Seeker:5B64:/
+4307.1 "Mercy Fourfold 1" #sync / 1[56]:Trinity Seeker:5B94:/
+4309.0 "Mercy Fourfold 2" #sync / 1[56]:Trinity Seeker:5B94:/
+4309.7 "Seasons Of Mercy" sync / 1[56]:Seeker Avatar:5AC7:/
+4311.0 "Mercy Fourfold 3" #sync / 1[56]:Trinity Seeker:5B94:/
+4312.9 "Mercy Fourfold 4" #sync / 1[56]:Trinity Seeker:5B94:/
+4314.3 "Merciful Breeze" sync / 1[56]:Trinity Seeker:5AC8:/
+4315.8 "Merciful Moon" sync / 1[56]:Aetherial Orb:5AC9:/
+4318.4 "Merciful Breeze" sync / 1[56]:Trinity Seeker:5AC8:/
+4321.8 "Merciful Blooms" sync / 1[56]:Trinity Seeker:5ACA:/
+4325.1 "--chains--" duration 3.1 # sync / 00:0000:0080:/
```

Comment:
:eyes:

Reply:
I uh. I didn't account for tether syncs existing in timelines *at all*. This *should* have been shoved through to manual fixes, [but this one *just so happened* to match the existing `gameLog` regex logic](https://regex101.com/r/WPwaRD/1)